### ISSUE TITLE
Add ability to add custom twig filters

### ISSUE THREAD

**Issue by ddimitrioglo** [ddimitrioglo/web-boost#20]

**Comment by ddimitrioglo** [comment]
Maybe something like this?

```js
module.exports = {
  extendFilter: {
    backwords: (value) => {
      return value.split(' ').reverse().join(' ');
    }
  },
  extendFunction: {
    repeat: (value, times) => {
      return new Array(times + 1).join(value);
    }
  }
};
```

**Comment by ddimitrioglo** [comment]
Implemented in `web-boost@1.4.0`
